<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureXCPOverEthernetDialog</class>
 <widget class="QDialog" name="ConfigureXCPOverEthernetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XCP over ethernet configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons.qrc">
    <normaloff>:/Icons/car.png</normaloff>:/Icons/car.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ConfigureXCPOverEthernetWidget" name="tabConnection_0">
      <attribute name="title">
       <string>Connection (0)</string>
      </attribute>
     </widget>
     <widget class="ConfigureXCPOverEthernetWidget" name="tabConnection_1">
      <attribute name="title">
       <string>Connection (1)</string>
      </attribute>
     </widget>
     <widget class="ConfigureXCPOverEthernetWidget" name="tabConnection_2">
      <attribute name="title">
       <string>Connection (2)</string>
      </attribute>
     </widget>
     <widget class="ConfigureXCPOverEthernetWidget" name="tabConnection_3">
      <attribute name="title">
       <string>Connection (3)</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigureXCPOverEthernetWidget</class>
   <extends>QWidget</extends>
   <header>ConfigureXcpOverEthernetWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureXCPOverEthernetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureXCPOverEthernetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
